ch01.asciidoc

```diff
@@ -43,11 +43,15 @@ We'll be seeing them throughout this book:
 consumer::
 A regular consumer on the Internet or in the offline world who wants to make purchases.
 
+Alice runs a small coffee shop in an upmarket street in Berlin.  She knows about bitcoin and wants to accept it in her shop, but has been reluctant to do so because she knows that bitcoin payments take approx. 10 minutes to be confirmed into her account, and that the fees that her customer needs to pay for a bitcoin payment are variable.  With the lightning network, she knows that her regular clients, such as Beppe can pay for their coffee at her shop, and she knows that the bitcoin is instantly in her wallet, and that the fees for her clients are small enough not to concern her clients. 
+
 content creator / curator::
 A person or platform offering content on the web.
 They want to install a pay wall or get tipped by their fans and consumers.
 This could even include music or video streaming on demand paying in real time
 
+Dennis is a 9 year old boy in Australia, who wanted to buy a games console like his friends.  His dad told him that he had to earn the money himself.  Dennis is an aspiring artist, but knows that while he is still learning, he can't charge very much for his artwork.  After learning about bitcoin, he setup a website to sell his drawings across the internet. By using the lightning network, Dennis was able to charge $1 for one of his drawings.  By being able to set a fair price (which would normally be considered a micropayment) and use a global currency such as bitcoin, Dennis was able to sell his art work around the world and buy the games console he dreamed of. 
```

Comment:
hmmm this sounds so familiar (: I like the idea of including use cases that have happened c.f.: 
https://bitcoinist.com/bitcoin-pictures-lightning-network/ 

However I am not sure if this might result in legal / moral /ethical issues. Anyone any ideas?

Reply:
Ha, you spotted it!  Dennis gives his approval to be used as a use case  (and can provide proof if necc :) - The use case (global micropayments & serving the unbanked) is valid, but if basing it on real life examples deflects attention from the core message, then lets change/ rework it.

Reply:
check #28  and please continue the discussion there.